src/Main.java

```diff
@@ -1,7 +1,15 @@
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) throws InterruptedException {
         Person person = new Person(22, 178, "alb", new Clock(3));
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter name: ");
+        scanner.nextLine();
+        System.out.println("Enter age: ");
+        scanner.nextInt();
+
```

Comment:
The person should be build based on the data you just entered.
Consider the name should be put in the properties field, and height in the height field.